bitnami/mariadb-galera/values-production.yaml

```diff
@@ -186,10 +186,10 @@ ldap:
   ## Enable LDAP support
   ##
   enabled: false
-  uri:
-  base:
-  binddn:
-  bindpw:
+  uri: ""
```

Comment:
I'm curious, is there a linter or something for things like this?

Reply:
This specific behaviour escapes linter's control